### ISSUE TITLE
Dismiss confirmation dialog after selecting action

### ISSUE THREAD

**Issue by ponomarevlm** [ahe/2dc_jqgrid#12]
One can hit "Delete" twice in confirmation dialog and in second time get an error.

**Comment by kritik** [comment]
yes, there is a problem. The problem is that dialogs is not closing after deleting/adding/editing. I'm trying to find the solution
